### PR TITLE
Promises gamecontroller

### PR DESCRIPTION
**Pull request by JamesAdamHughes**
Used some promises where they weren't needed because why the hell not.

### CODE REVIEW

**Comment by JamesAdamHughes** [comment]
Remember y'all to run npm install.

Hey, that rhymes...

File: test/e2e/afterRoundSpec.js [hunk 1]
```diff
@@ -51,14 +51,14 @@ describe('After each round', function() {
...
 	});
 
-	it('can see who voted for what answers', function() {
+	// it('can see who voted for what answers', function() {
 
-		// expect(firstClonageUser.element.all(by.repeater('result in getPlayerRoundResults()')).get(0).element(by.binding('result.playersWhoVotedForThis.length')).getText()).toContain('1');
-		// expect(firstClonageUser.element.all(by.repeater('result in getPlayerRoundResults()')).get(1).element(by.binding('result.playersWhoVotedForThis.length')).getText()).toContain('1');
-		// expect(secondClonageUser.element.all(by.repeater('result in getPlayerRoundResults()')).get(0).element(by.binding('result.playersWhoVotedForThis.length')).getText()).toContain('1');
-		// expect(secondClonageUser.element.all(by.repeater('result in getPlayerRoundResults()')).get(1).element(by.binding('result.playersWhoVotedForThis.length')).getText()).toContain('1');
+	// 	expect(firstClonageUser.element.all(by.repeater('result in getPlayerRoundResults()')).get(0).element(by.binding('result.playersWhoVotedForThis.length')).getText()).toContain('1');
```
Comment (Geor23) [inline test/e2e/afterRoundSpec.js]: ?

Comment (JamesAdamHughes) [inline test/e2e/afterRoundSpec.js]: these were commented out originally

**Comment by BRHoward** [comment]
acceptable

**Pull request merged**